packages/netlify-cms-lib-util/src/APIUtils.ts

```diff
@@ -26,11 +26,11 @@ export function generateContentKey(collectionName: string, slug: string) {
 
 export function parseContentKey(contentKey: string) {
   const index = contentKey.indexOf('/');
-  return { collection: contentKey.substr(0, index), slug: contentKey.substr(index + 1) };
+  return { collection: contentKey.slice(0, index), slug: contentKey.slice(index + 1) };
```

Comment:
`index` could possibly be negative (-1) so to have the same result as `substr()` we either have to check for that or use `substring` on `contentKey.slice(0, index)`

Reply:
It's expected for `contentKey` to always include a `/` (it's the branch name, e.g. `cms/...`